Super surprising got a sqlite3 specific failure

```
FAILED test_app/tests/rbac/api/test_access_assignment_uuid_lookup.py::TestAccessListRelatedLinksWithAnsibleID::test_team_access_list_related_links_use_ansible_id - AssertionError: assert not True
 +  where True = <built-in method endswith of str object at 0x7fe5f4ef32b0>('1/')
 +    where <built-in method endswith of str object at 0x7fe5f4ef32b0> = '/api/v1/role_team_access/aap.inventory/1/5ca295bc-9064-47da-8983-7e1ed5592d31/'.endswith
= 1 failed, 2103 passed, 25 skipped, 13 xfailed, 42 warnings in 323.82s (0:05:23) =
py311sqlite: exit 1 (328.71 seconds) /home/runner/work/django-ansible-base/django-ansible-base> pytest -n auto --cov=. --cov-report=xml:coverage.xml --cov-report=html --cov-report=json --cov-branch --junit-xml=django-ansible-base-test-results.xml test_app/tests pid=3955
py311sqlite: docker> remove '113dcd32ace8' (from 'db')
```


Ooooooh! I see it! This is numerology flake. The UUID string _just happens to_ end with the same decimal number as the primary key.

```
E        +    where <built-in method endswith of str object at 0x7fe5f4ef32b0> = '/api/v1/role_team_access/aap.inventory/1/5ca295bc-9064-47da-8983-7e1ed5592d31/'.endswith
```

Because we asserted that it does not end with `"1/"`, and it does! Because the uuid4 ends with 1 for this one single run. That will fail 1 out of every 10 times.

--- a/test_app/tests/rbac/api/test_access_assignment_uuid_lookup.py
+++ b/test_app/tests/rbac/api/test_access_assignment_uuid_lookup.py
@@ -152,7 +152,7 @@ class TestAccessListRelatedLinksWithAnsibleID:
         assert expected_ansible_id in team_data['related']['details']
         # Check that the URL ends with the ansible_id, not the primary key
         assert team_data['related']['details'].endswith(f'{expected_ansible_id}/')
-        assert not team_data['related']['details'].endswith(f'{team.pk}/')
+        assert not team_data['related']['details'].endswith(f'/{team.pk}/')
 
     def test_user_access_list_fallback_to_pk_when_no_resource(self, admin_api_client, inventory, inv_rd):
         """Test fallback to primary key when user has no resource (edge case)"""

